lib/galaxy/job_execution/output_collect.py

```diff
@@ -479,10 +479,12 @@ def walk_over_extra_files(target_dir, extra_file_collector, job_working_director
     if os.path.isdir(directory):
         for filename in os.listdir(directory):
             path = os.path.join(directory, filename)
-            if os.path.isdir(path) and extra_file_collector.recurse:
-                # The current directory is already validated, so use that as the next job_working_directory when recursing
-                for match in walk_over_extra_files(filename, extra_file_collector, directory, matchable):
-                    yield match
+            log.error("%s isdir %s recurse %s" % (path, os.path.isdir(path), extra_file_collector.recurse))
```

Comment:
Can you drop this - or make it debug if it is important to keep?

Reply:
Sure.